app/assets/javascripts/client/home/home_anon.html.erb

```diff
@@ -1,103 +1,66 @@
-<section class='hero'>
-  <div class='row'>
-    <div class='col-sm-12 text-center'>
-      <h1>PDredesign</h1>
-    </div>
-  </div>
-  <div class='row'>
-    <div class='col-sm-12 text-center'>
-      <p>A digital toolkit and community of practice</br>
-        dedicated to improving professional</br>
-        development for teachers.</p>
-    </div>
+<div class="hero-header">
+  <div class="hero-banner">
+    <h1>PDredesign</h1>
+    <p class="text-center">A digital toolkit and community of practice dedicated to improving
+      <u>professional development</u> for teachers.</p>
```

Comment:
Fair point.

I did look for a CSS class since I don't particularly like the use of `<u>`, but this also follows suit from [Bootstrap's own recommendation](http://getbootstrap.com/css/#type-inline-text).

I can change it since I don't particularly agree with `<u>` anyway.